### ISSUE TITLE
Stripe payment action not executing

### ISSUE THREAD

**Issue by Sistemas-Tecsellent** [FlutterFlow/flutterflow-issues#225]
Issue tracker is **ONLY** used for reporting bugs. New feature suggestions and questions should be discussed on Community or submitted through our user feedback form.

Your issue may already be reported! Please search in the [issue tracker](../) before creating one.

Please **thumbs up** this issue if you have also experienced it. You may also add more information if there is something relevant that was not mentioned. However, please refrain from comments that are not constructive, like "I have this problem too", etc.

## Expected behavior (required)

The user clicks a button and the bottom sheet to enter the card info appears.

## Current behavior (required)

Nothing happens. After running it locally on verbose mode these are the message when I click the button. I already configured the right API keys and the same happens both when marked as production and testing. I even wrote directly the values to make sure it was not because a null value, the same happens.

[  +18 ms] W/DynamiteModule( 4496): Local module descriptor class for com.google.android.gms.providerinstaller.dynamite notfound.
[  +26 ms] I/DynamiteModule( 4496): Considering local module com.google.android.gms.providerinstaller.dynamite:0 and remotemodule com.google.android.gms.providerinstaller.dynamite:0
[   +1 ms] W/ProviderInstaller( 4496): Failed to load providerinstaller module: No acceptable module
com.google.android.gms.providerinstaller.dynamite found. Local version is 0 and remote version is 0.
[ +332 ms] I/flutter ( 4496): Cloud call error!
[        ] I/flutter ( 4496): Code: unauthenticated
[        ] I/flutter ( 4496): Details: null
[        ] I/flutter ( 4496): Message: UNAUTHENTICATED

## To Reproduce (required)

1. Add the stripe payment action to a button
2. Run the app on a mobile
3. Nothing happens

## Context (required)

Trying to let a user pay for a product

## Your environment

* Version of FlutterFlow used: 
* Platform (e.g. Web, MacOS Desktop): 
* Browser name and version: 
* Operating system and version (desktop or mobile):

Flutter 3.3.9
Flutter doctor shows no issues.
Tested on Pixel 3XL

**Comment by bulgariamitko** [comment]
Can you send an image of your Action and all the parameters?

**Comment by Sistemas-Tecsellent** [comment]
Thanks for the help. I fixed it. For what I can say the issue was in two places.

The generate stripe payment cloud function had no one assigned as invoker, I suppose it couldn't be called by the app because of this.

Second. It seems that after writing my API keys on flutterflow they were not added on the commit to the github repo and thus couldn't be tested. I deleted -> commit -> added them -> commit and repeat this several times until I saw them on the commit changes
